app/Models/User.php

```diff
@@ -198,4 +199,14 @@ public function userTokens()
     {
         return $this->hasMany(UserToken::class);
     }
+
+    /**
+     * Get the preferred locale of the entity.
+     *
+     * @return string|null
+     */
+    public function preferredLocale()
```

Comment:
you could use an attribute for that instead of a method, if that helps.

Reply:
Actually this comes from the `HasLocalePreference` interface, and just return a column value.